tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/NLogTests.cs

```diff
@@ -138,16 +151,19 @@ public static IEnumerable<object[]> GetTestDataDirectSubmission()
                             continue; // pre 4.0.0 doesn't have JSON support
                         }
 
-                        yield return item.Concat(agentless).Concat(LoggingContext.None).Concat(configType);
+                        yield return item.Concat(agentless).Concat(LoggingContext.None).Concat(configType).Concat(Enable128BitInjection.Enable);
+                        yield return item.Concat(agentless).Concat(LoggingContext.None).Concat(configType).Concat(Enable128BitInjection.Disable);
```

Comment:
nit: Maybe it's time we ditched the horrible Concat that creates 4 different arrays 😅 
```suggestion
                        yield return [item[0], agentless, LoggingContext.None, configType, Enable128BitInjection.Enable];
                        yield return [item[0], agentless, LoggingContext.None, configType, Enable128BitInjection.Disable];
```

Reply:
I agree, I will plan to do this along with some other improvements here in a follow up PR